components/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/servlet/SAMLSSOProviderServlet.java

```diff
@@ -2234,4 +2246,18 @@ private void sendNotificationForECPRequest(HttpServletResponse resp, String erro
         }
     }
 
+    /**
+     * To retrieve session context of the given session identifier.
+     *
+     * @param sessionIdentifier Session identifier.
+     * @param loginTenantDomain Login tenant domain.
+     * @return Session context for the given session identifier.
+     */
+    private SessionContext getSessionContext(String sessionIdentifier, String loginTenantDomain) {
+
+        if (sessionIdentifier != null && loginTenantDomain != null) {
```

Comment:
use StringUtils and check for empty and null

Reply:
Fixed